pkgs/tools/X11/go-sct/default.nix

```diff
@@ -20,7 +21,7 @@ buildGoPackage rec {
   meta = with stdenv.lib; {
     description = "Color temperature setting library and CLI that operates in a similar way to f.lux and Redshift";
     license = licenses.mit;
-    maintainers = with maintainers; [ cstrahan ];
-    platforms = platforms.linux;
+    maintainers = with maintainers; [ rvolosatovs cstrahan ];
+    platforms = platforms.linux ++ platforms.windows;
```

Comment:
How do I test the windows build btw?

Reply:
You can install Nix to Cygwin. It didn't work very well the last time I tried, though :-(

Reply:
Also, Go packages have default `meta.platforms`, so it might make sense to remove it entirely: https://github.com/NixOS/nixpkgs/blob/7d238e6da8a2146ab3c76d52b5ec2fd6dd59444e/pkgs/development/go-modules/generic/default.nix#L224

Reply:
FTR, I added this simply because the package itself has windows support. I do not have a windows machine to test this on.
@yegortimoshenko: this package does not support darwin.